tests/unit/reconcile/query_builder/test_threshold_query.py

```diff
@@ -19,7 +19,7 @@ def test_threshold_comparison_query_with_one_threshold(table_conf_with_opts, tab
     table_schema, _ = table_schema
     table_schema.append(Schema("s_suppdate", "timestamp"))
     comparison_query = ThresholdQueryBuilder(
-        table_conf, table_schema, "source", get_dialect("oracle")
+        table_conf, table_schema, "target", get_dialect("databricks")
```

Comment:
Why change? I wanted to test that It works the same irrespective of layer and source. 
test_threshold_comparison_query_with_dual_threshold test case covers for target and databricks.

Reply:
Yes.But as per logic data resides in Databricks views all the data types are related to Databricks.So we wanted to restrict to Databricks and Target.

Reply:
No need .. It works as a negative case also. I suggest revert it back.

Reply:
done.